proto/table.proto

```diff
@@ -148,6 +148,17 @@ message ImportExportShared {
     }
     repeated string dedupColumnsNames = 6; // columns on which base will be deduplication performed
   }
+  // TODO dopsat dokumentaci
+  message ExportOptions {
+    bool isCompressed = 1;
+    google.protobuf.Any meta = 2;
+    message TeradataExportOptionsMeta {
+      string bufferSize = 1;
+      string maxObjectSize = 2;
+      bool dontSplitRows = 3;
+      bool singlePartFile = 4;
```

Comment:
Bavili sme sa o tom, že tady ty options propagovat nebudeme, mělo by to byt staticky třeba 50MB maximálně by to mohlo byt přímo v ExportOptions něco jako maxSliceSize nebo tak něco

Reply:
jj, dal jsem tam default 50 MB.

O nejakym predani ty max. velikosti jsem dost uvazoval, pac se bez toho dost blbe psaly testy se slices... Ale muselo by to zase byt v Meta, pac ostatni backendy to nepodporujou:
```
  message ExportOptions {
    bool isCompressed = 1;
    google.protobuf.Any meta = 2;
    message TeradataExportOptionsMeta {
      string maxSliceSize = 1;
    }
  }
```

Reply:
Tak jiné backendy v driveru nemáme zatím takže by to mohla byt zrovna ExportOptions prop, ale asi bych to teď neřesil.